moirae/models/utils.py

```diff
@@ -0,0 +1,12 @@
+"""
+Collection of utility functions and classes for the models
+"""
+from .base import DegradationModel, HealthVariable
+
+
+class DummyDegradation(DegradationModel):
+    """
+    Class corresponding to no degradation, that is, the updated A-SOH is the same as the previous A-SOH.
+    """
+    def update_asoh(self, previous_asoh: HealthVariable, *args, **kwargs) -> HealthVariable:
```

Comment:
Could you copy the function signature from the base class? It should not change for subclasses.

Reply:
Sorry, reverted to bad habits here. Fixing it!